### PR TITLE
OCPBUGS-38025: Add Network Observability Operator must-gather link to must-gather page listing

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OCPBUGS-38025
Link to docs preview:

https://84384--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by msherif1234** [comment]
do u need to add usage section too https://github.com/netobserv/must-gather?tab=readme-ov-file#usage ?

**Comment by skrthomas** [comment]
@coffeegoesincodecomesout can you PTAL at this and let me know if it is good for the bug you filed?

File: modules/gathering-data-specific-features.adoc [hunk 1]
```diff
@@ -45,6 +45,9 @@ endif::openshift-rosa,openshift-dedicated[]
 |`registry.redhat.io/openshift-logging/cluster-logging-rhel9-operator:v<installed_version_logging>`
 |Data collection for {logging}.
 
+|`image=quay.io/netobserv/must-gather`
```
Comment (Amoghrd) [inline modules/gathering-data-specific-features.adoc:48]: I dont know if the `image=` is required. I see that the table description mentions to pass this image with the `--image` option

Comment (skrthomas) [reply modules/gathering-data-specific-features.adoc:48]: Oh great catch!

**Review by jeana-redhat — APPROVED** [review]
LGTM :shipit:

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**